Sming/Libraries/MFRC522/MFRC522.cpp

```diff
@@ -0,0 +1,1767 @@
+/*
+* MFRC522.cpp - Library to use ARDUINO RFID MODULE KIT 13.56 MHZ WITH TAGS SPI W AND R BY COOQROBOT.
+* NOTE: Please also check the comments in MFRC522.h - they provide useful hints and background information.
+* Released into the public domain.
+*/
+#include <user_config.h>
+#include <SmingCore/SmingCore.h>
```

Comment:
Last, but not least, you should make the code to compile. I would recommend you to move these
includes 
```
#include <user_config.h>
#include <SmingCore/SmingCore.h>
```

from here and put them into the header file `MFRC522.h` as 
```
#include <user_config.h>
#include <SmingCore.h>
```

Commit the changes and make sure that all tests pass.